.circleci/config.yml

```diff
@@ -140,3 +148,41 @@ jobs:
       - run:
           name: Visual Tests
           command: yarn test:visual
+
+  e2e:
+    docker:
+      - image: cypress/base:8
+    environment:
+      TZ: "/usr/share/zoneinfo/America/Los_Angeles"
+    steps:
+      - run:
+          name: Update yarn
+          command: |
+            # remove default yarn
+            rm -rf $(dirname $(which yarn))/yarn*
+            # download latest
+            rm -rf ~/.yarn
+            curl -o- -L https://yarnpkg.com/install.sh | bash
+            echo 'export PATH="${PATH}:${HOME}/.yarn/bin"' >> $BASH_ENV
+      - checkout
+      # because we don't invoke npm (we use yarn) we need to add npm bin to PATH manually
+      - run:
+          name: Add npm bin to PATH
+          command: echo 'export PATH="${PATH}:$(npm bin)"' >> $BASH_ENV
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "yarn.lock" }}
```

Comment:
This changed to `v1.1` in master, please update it here (and in `save_cache`) as well

Reply:
thanks for pointing out, changed